config/initializers/hypershield.rb

```diff
@@ -0,0 +1,18 @@
+Hypershield.schemas = {
+  hypershield: {
+    hide: %w[
+      auth_data_dump
```

Comment:
All in all let's just be super thorough in coming up with anything that _might_ be sensitive here.

Reply:
A while back we ran a contest and gathered this info...

```
    t.string "shipping_address"
    t.string "shipping_address_line_2"
    t.string "shipping_city"
    t.string "shipping_company"
    t.string "shipping_country"
    t.string "shipping_name"
    t.string "shipping_postal_code"
    t.string "shipping_state"
    t.boolean "shipping_validated", default: false
    t.datetime "shipping_validated_at"
```

These days we do anything like that through Shopify and don't need any of this info. It _would_ be sensitive, but we also could just remove those columns we don't use anymore.

Reply:
+1 for removing the columns we dont use that way there is no question that we are NOT collecting that information

Reply:
#6477 has been merged, shipping columns are gone, albeit roughly done 😬